platforms/corda-ent/charts/signer/templates/deployment.yaml

```diff
@@ -170,15 +165,15 @@ spec:
           mountPath: /opt/corda/DATA
         resources:
           requests:
-            memory: {{ .Values.cordaJarMx }}G
+            memory: {{ .Values.config.cordaJar.resources.requests }}
```

Comment:
As you already have a separate cordaJar.memorySize, this can be pod.resources as these are not related to cordaJar but rather with pods